DSSE/environment/person.py

```diff
@@ -5,6 +5,8 @@
 
 class Person():
     """
+    Class that represents a shipwrecked person in the environment.
+
     Attributes:
     -----------
```

Comment:
It seems that this docstring is outdated.

Reply:
You are right! My mistake. Just removed the docstring. Who uses docstrings, anyway?